.github/workflows/stale.yml

```diff
@@ -0,0 +1,35 @@
+# https://docs.github.com/en/actions
+
+name: "Issues and pull requests"
+
+on:
+  schedule:
+    - cron: "0 12 * * *"
+
+env:
+  DAYS_BEFORE_CLOSE: 90
+  DAYS_BEFORE_STALE: 1095
```

Comment:
* 3 years to be considered as stale? I would change it to 365 and even that is too much probably
* I believe we need different values for issues and for PRs. Issues may be valid after longer time, but PRs need to be updated frequently to be up-to-date with main branch